docs/configuration.rst

```diff
@@ -69,3 +69,9 @@ By default ``tr_case`` is set to::
 
 Please read :ref:`tr_file` for more details.
 
+tr_report_template
+------------------
+
+``tr_report_template`` allows to specify a custom template for testcase visualisation. Provide a filename and place the
+file in the same directory as your conf.py (or provide an absolute path). For an example with just a changed order uncomment
+``tr_report_template = "custom_test_report_template.txt"`` in conf.py
```

Comment:
> For an example with just a changed order uncomment
``tr_report_template = "custom_test_report_template.txt"`` in conf.py

The `conf.py` is only available in the github repo, in which a user normal does not take a look.
Or in worst case, the user things ``tr_report_template`` is already set in his/her project conf.py.

If you want to reference/show an example, I would integrate the file as code-block here:
``.. literalinclude:: file.template``.